### ISSUE TITLE
Project details could not be queried.

### ISSUE THREAD

**Issue by zq1997** [moritzgloeckl/overleaf-sync#11]
I'm sorry for just provide such little information, but there seems no verbose option provided.
```
Project queried successfully.
✅  Querying project
Error: Project details could not be queried.
```
New installed from pip, python 3.8.2, login successfully, but error when sync.
Reproduced with different project and machine.

**Comment by zq1997** [comment]
I debugged it out.
The reason is that I install `socketIO-client` from pip, so it is the newest version `0.7.2`
But This project only works with `0.5.7.2`

Thus this code failed to construct a SocketIO instance.
```
# olclient.py:154
        socket_io = SocketIO(
            BASE_URL,
            params={'t': int(time.time())},
            headers={'Cookie': cookie}
        )
```
Maybe some modification should be made to work with newer `socketIO-client` version.
Further more, how about drop the usage of `SocketIO` ? I found `socketIO-client` is only used in this code, and I remember that pure Python can also do socket communication, drop `socketIO-client` and use standard python lib may cause less dependency problem.

Also, when I pip install this project, it not install  `socketIO-client` automatically as an dependecy. Is this a bug?

**Comment by moritzgloeckl** [comment]
Hi @zq1997 !

Thank you for your bug report. Let me clarify:

- Yes this is a bug I need to add the `socketIO-client` dependency so it will be installed via pip. Working on this now.
- It needs this specific version because Overleaf is working with an "outdated" Socket.IO version. This is the last version of the Python library to support the version their using.
- You surely can implement it without the socketIO client, but beware that we're talking about Socket.IO Protocol and not just simple Sockets or Websocket. I found it easier to use the library then to implement the Socket.IO handshake myself. Feel free to open a PR though.

**Comment by moritzgloeckl** [comment]
Fixed with #12
